keras_nlp/models/xlm_roberta/xlm_roberta_preprocessing.py

```diff
@@ -21,17 +21,150 @@
     RobertaMultiSegmentPacker,
 )
 from keras_nlp.tokenizers.sentence_piece_tokenizer import SentencePieceTokenizer
+from keras_nlp.utils.python_utils import classproperty
 from keras_nlp.utils.tf_utils import tensor_to_string_list
 
 
+@keras.utils.register_keras_serializable(package="keras_nlp")
+class XLMRobertaTokenizer(SentencePieceTokenizer):
+    """XLM-RoBERTa tokenizer layer based on the SentencePiece subword tokenizer.
```

Comment:
XLM-RoBERTa tokenizer layer based on SentencePiece.

SentencePiece is technically an everything tokenizer (not just subword).